Main.elm

```diff
@@ -225,31 +433,19 @@ formView model =
         , Html.div []
             [ Html.text model.error ]
         , Html.div []
-            [ Html.label [ Html.Attributes.for "email" ] [ Html.text "Email" ]
-            , Html.input
-                [ Html.Attributes.id "email"
-                , Html.Attributes.type_ "text"
-                , Html.Attributes.placeholder "joe.bart@team.tld"
-                , Html.Attributes.value model.email
-                , Html.Events.onInput NewEmail
-                ]
-                []
-            ]
-        , Html.div []
-            [ Html.label [ Html.Attributes.for "password" ] [ Html.text "Passphrase" ]
-            , Html.input
+            [ Html.input
```

Comment:
You should add the `form-control` class to benefit from bootstrap styles for this text input.